gr-qtgui/grc/qtgui_freq_sink_x.block.yml

```diff
@@ -39,6 +39,13 @@ parameters:
     option_labels: [Blackman-harris, Hamming, Hann, Blackman, Rectangular, Kaiser,
         Flat-top]
     hide: part
+-   id: norm_window
+    label: Normalize Window Power
+    dtype: enum
+    default: 'False'
+    options: ['True', 'False']
```

Comment:
The change above changes the string values (`'True'`) to type values (`True`). Is this required here as well?